doc/operators/ledger.rst

```diff
@@ -0,0 +1,53 @@
+Ledger
+======
+
+All network state is contained in the ledger. A single up-to-date copy of the ledger is enough to start a successor
+service if necessary, following the :ref:`operators/recovery:Catastrophic Recovery` procedure.
+
+Each node in a network creates and maintains its own local copy of the ledger. Committed entries are always identical,
+but a node may be more or less up to date, and uncommitted entries may differ.
+
+Location
+--------
+
+Nodes can be configured to store their ledger under a particular directory with the `--ledger-dir` command-line option.
+
+File layout
+-----------
+
+The ledger directory contains a series of files. File size is controlled by the ``--ledger-chunk-max-bytes`` command line switch.
```

Comment:
s/switch/argument as "switch" sounds like a flag?

Reply:
yeah, although I think "option" may be right here.